<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BrowseFilesWindow</class>
 <widget class="QMainWindow" name="BrowseFilesWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>339</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Browse Files</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources/icons.qrc">
    <normaloff>:/myappicons/duetto_logo.png</normaloff>:/myappicons/duetto_logo.png</iconset>
  </property>
  <property name="toolTip">
   <string>The browse window allow to visualize several signals and its properties. You can select one or many to open in sound Lab.</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="9">
     <widget class="QPushButton" name="playFile_bttn">
      <property name="toolTip">
       <string>Play the first of the selected signals.</string>
      </property>
      <property name="text">
       <string>Play</string>
      </property>
      <property name="icon">
       <iconset resource="resources/icons.qrc">
        <normaloff>:/myappicons/02049_26x26.png</normaloff>:/myappicons/02049_26x26.png</iconset>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="9">
     <widget class="QLineEdit" name="folderPath_lineEdit">
      <property name="toolTip">
       <string>Input folder of signals to browse.</string>
      </property>
      <property name="text">
       <string>Seleccione la carpeta</string>
      </property>
     </widget>
    </item>
    <item row="0" column="9">
     <widget class="QPushButton" name="folderPath_bttn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Explore folders to add signals to the currently visualized on table.</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" rowspan="2" colspan="10">
     <widget class="QTableWidget" name="files_tablewidget">
      <property name="toolTip">
       <string>Table with parameters of signals to analize them.</string>
      </property>
      <column>
       <property name="text">
        <string>Filename</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Size</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Creation Date</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Duration</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="4" column="8">
     <widget class="QPushButton" name="fileDown_bttn">
      <property name="toolTip">
       <string>Select next signal. (Next to currently selected)</string>
      </property>
      <property name="text">
       <string>File Down</string>
      </property>
      <property name="icon">
       <iconset resource="resources/icons.qrc">
        <normaloff>:/myappicons/downfile.png</normaloff>:/myappicons/downfile.png</iconset>
      </property>
     </widget>
    </item>
    <item row="4" column="7">
     <widget class="QPushButton" name="fileUp_bttn">
      <property name="toolTip">
       <string>Select previous signal. (Previous to currently selected)</string>
      </property>
      <property name="text">
       <string>File Up</string>
      </property>
      <property name="icon">
       <iconset resource="resources/icons.qrc">
        <normaloff>:/myappicons/upfile.png</normaloff>:/myappicons/upfile.png</iconset>
      </property>
     </widget>
    </item>
    <item row="4" column="6">
     <widget class="QPushButton" name="openFile_bttn">
      <property name="toolTip">
       <string>Open selected signals of the table on sound Lab</string>
      </property>
      <property name="text">
       <string>Open</string>
      </property>
      <property name="icon">
       <iconset resource="resources/icons.qrc">
        <normaloff>:/myappicons/81.ico</normaloff>:/myappicons/81.ico</iconset>
      </property>
     </widget>
    </item>
    <item row="4" column="5">
     <widget class="QPushButton" name="invertSelection_bttn">
      <property name="toolTip">
       <string>Invert selection of signals on table.</string>
      </property>
      <property name="text">
       <string>Invert</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QPushButton" name="selectAll_bttn">
      <property name="toolTip">
       <string>Select all signals on table</string>
      </property>
      <property name="text">
       <string>Select All</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionFileUp">
   <property name="text">
    <string>FileUp</string>
   </property>
  </action>
  <action name="actionFileDown">
   <property name="text">
    <string>FileDown</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="text">
    <string>Play</string>
   </property>
  </action>
  <action name="actionAddFileButton">
   <property name="text">
    <string>AddFileBttn</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="text">
    <string>SelectAll</string>
   </property>
  </action>
  <action name="actionInvertSelection">
   <property name="text">
    <string>InvertSelection</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>playFile_bttn</sender>
   <signal>clicked()</signal>
   <receiver>actionPlay</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>303</x>
     <y>297</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fileDown_bttn</sender>
   <signal>clicked()</signal>
   <receiver>actionFileDown</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>218</x>
     <y>297</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fileUp_bttn</sender>
   <signal>clicked()</signal>
   <receiver>actionFileUp</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>133</x>
     <y>297</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>openFile_bttn</sender>
   <signal>clicked()</signal>
   <receiver>actionOpen</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>297</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>folderPath_bttn</sender>
   <signal>clicked()</signal>
   <receiver>actionAddFileButton</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>448</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selectAll_bttn</sender>
   <signal>clicked()</signal>
   <receiver>actionSelectAll</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>297</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>invertSelection_bttn</sender>
   <signal>clicked()</signal>
   <receiver>actionInvertSelection</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>355</x>
     <y>297</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
